internal/cli/atlas/clusters/update.go

```diff
@@ -126,14 +126,8 @@ func UpdateBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "update [clusterName]",
 		Short: "Update a MongoDB Atlas cluster.",
-		Example: fmt.Sprintf(`  Update tier for a cluster:
```

Comment:
the problem with this approach is that the help of the command now looses the extra examples (--help will no longer be able to show them) 
I think that's a big drawback if you ask me but I think it needs to be discussed with @boooczek, sacrifice examples on the terminal for examples on the web.
If @boooczek thinks this is ok then don't mind me, if he doesn't one alternative we can work on the cobra2snooty side of things is parsing this string into multiple examples by some sort of break

Reply:
Ah, I actually did not realize that. I will check with @boooczek.

Reply:
Update: now that changes to cobra2snooty have merged, I pushed a change in this PR that puts back the multi-examples in this field so that they all appear for --help. Now cobra2snooty breaks out examples using the # symbol and put them in their own code blocks.